recipe/meta.yaml

```diff
@@ -40,13 +39,16 @@ test:
   requires:
     - pip
     - pytest >=5.0.1
+    - pytest-cov
     - pytest-timeout
-    - pytest-xdist
+    - flake8
+    - black
+    - mypy
```

Comment:
```suggestion

```
These are dev dependencies, used for linting and such. You can safely remove them.